.swift-format

```diff
@@ -0,0 +1,52 @@
+{
+  "blankLineBetweenMembers" : {
+    "ignoreSingleLineProperties" : true
+  },
+  "indentation" : {
+    "spaces" : 4
+  },
+  "lineBreakBeforeControlFlowKeywords" : false,
+  "lineBreakBeforeEachArgument" : false,
+  "lineLength" : 160,
+  "maximumBlankLines" : 2,
```

Comment:
Does the maximumBlankLines setting count the invisible characters? Because if I count the number of returns (2), then the max blank lines is 1, which would explain why it deletes the second blank line in every file, between the import declarations and the file comment.

So if the format counts in that manner, should `maximumBlankLines` be `3` ?